sources/core/Stride.Core.Design/Editor/ExternalEditors.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Stride contributors (https://stride3d.net)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Stride.Core.VisualStudio;
+
+namespace Stride.Core.Editor
+{
+    public static class ExternalEditors
+    {
+        private static Lazy<List<EditorInfo>> IDEInfos = new Lazy<List<EditorInfo>>(BuildIDEInfos);
+        
+        public static IEnumerable<EditorInfo> AvailableEditors => IDEInfos.Value;
```

Comment:
This property and/or its class could do with a more fitting name, we're specifically talking about IDEs here, not just any kind of text, model or image editor. We're in ``Stride.Core.Editor`` here so the distinction is fairly important.